### PR TITLE
feature: M3-929 Printable Invoices

### PR DESCRIPTION
**Pull request by rjstires**
re: https://github.com/linode/manager/pull/4327

## Description
Provide a printable invoice. Hey, they could even print it do PDF!

## Type of Change
- Feature

### CODE REVIEW

File: src/features/Billing/InvoiceDetail/InvoiceDetail.tsx [hunk 1]
```diff
@@ -107,25 +99,13 @@ class InvoiceDetail extends React.Component<CombinedProps, State> {
...
-                {this.renderContent()}
-              </TableBody>
-            </Table>
+            <InvoiceTable loading={loading} items={items} errors={errors} />
           </Grid>
           <Grid item xs={12}>
             <Grid container justify="space-between">
-              <Grid item className={classes.titleWrapper} />
+              <Grid item className={classes.titleWrapper}>
+                {invoice && <a target="_blank" href={`/account/billing/invoices/${invoice.id}/print`}>Download / Print</a>}
```
Comment (martinmckenna) [inline src/features/Billing/InvoiceDetail/InvoiceDetail.tsx]: should be a button for accessibility purposes

**Comment by rmcintosh** [comment]
May want to consider using a monospace font as the existing printable invoices do

File: src/features/Billing/InvoiceDetail/InvoiceDetail.tsx [hunk 2]
```diff
@@ -68,6 +68,18 @@ class InvoiceDetail extends React.Component<CombinedProps, State> {
...
+  pressPrint = (e:any) => {
+    const { invoice } = this.state;
+    const evtobj = window.event? event : e;
+    if (evtobj.keyCode === 80 && (evtobj.ctrlKey || evtobj.metaKey)) {
+      e.preventDefault();
+      return (
+        invoice && window.open(`/account/billing/invoices/${invoice.id}/print`, "_blank")
+      )
+    }
+    return false;
```
Comment (Jskobos) [inline src/features/Billing/InvoiceDetail/InvoiceDetail.tsx]: This logic is making the page un-tabbable

**Comment by rjstires** [comment]
https://github.com/linode/manager/pull/4327

**Pull request closed without merging**